PromoExchange/batman
# Carton upcharge old

See
https://trello.com/c/gTFC0gLK
### Test Plan
1. Ensure the following products have a populated price grid
   - Ensure Pretzel Acetate Trio
   - Small Gourmet Candy Box
   - Flip Top Tube with Imprinted Round Mints
   - Decorated Gourmet Pretzel Gift Box w Customized Label
2. Ensure prebid logs (attached) include shipping upcharge
   [choc_longboard_prebid.pdf](https://github.com/PromoExchange/batman/files/503811/choc_longboard_prebid.pdf)
